datasets/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .dataset_generic import Generic_WSI_Classification_Dataset, Generic_MIL_Dataset
+from .dataset_generic import save_splits, create_cv_folds
+from .preprocess_dataset import preprocess_sinai
```

Comment:
Could you change the preprocess_sinai to more generic.

Reply:
Given that datasets vary in data formats, it would be better to implement preprocessing functions customized for specific dataset rather than attempting to generalize them.